### DISCUSSION TITLE
Host _brand.yml online

### DISCUSSION THREAD

**Discussion by TeemuSailynoja** [posit-dev/brand-yml#57] [General]
Can I host a _brand.yml online, say in a Github repo, and specify a URL in the _quarto.yml of my different projects?

For example, if I have multiple dashboard projects that I would like to all update to the changes I make to my brand, when I run them the next time.

**Comment by mykolaskrynnyk** [comment]
As of now, it seems that you cannot specify a remote location for `_brand.yml` in Quarto. However, you can easily fetch a remote file to your project automatically using a pre-render command. See https://github.com/posit-dev/brand-yml/discussions/88#discussioncomment-13866138.
